plugins/powershell/help.md

```diff
@@ -11,7 +11,7 @@
 * For local Orchestrator execution, ensure connectivity to any network resources the script will use
 * For remote server execution, a PowerShell-enabled server and administrative credentials
 * When adding a password for the script, try to avoid special or escape characters such as single or double quotes
-* Round Robin is not supported
+* Currently we do not support the use of round-robin DNS lookups
```

Comment:
Suggestion - avoid use of 'we':
Note that round-robin DNS lookups are not supported.